pages/linux/tree.md

```diff
@@ -10,25 +11,25 @@
 
 `tree -d`
 
-- Print hidden files too:
+- Print hidden files too with colorization on:
 
-`tree -a`
+`tree -a -C`
 
 - Print the tree without indentation lines, showing the full path instead (use `-N` to not escape whitespace and special characters):
 
 `tree -i -f`
 
-- Print the size of each node next to it, in human-readable format:
+- Print the size of each node next to it, in human-readable format, with directories displaying their cumulative size (as in the `du` command):
```

Comment:
Wow, this is a pretty long description. Is this syncing with another copy of this page, or is it original?

If the latter, then if we could shorten this a bit and make it more concise that'd be great :-)

Same goes for the other particularly long one further down.

Reply:
This commit extended the description https://github.com/tldr-pages/tldr/commit/c294bdc4 in September 2016.

Reply:
@sbrl This commit extended the description https://github.com/tldr-pages/tldr/commit/c294bdc4 in September 2016.

Reply:
I would simplify it somewhere around this:

```suggestion
- Print the size of each file and the cumulative size of each directory, in human-readable format:
```
What do you think?